example/eventsource-polyfill.js

```diff
@@ -7430,7 +7432,11 @@ function EventSource (url, eventSourceInitDict) {
     event.delayMillis = delay
     _emit(event)
 
-    setTimeout(function () {
+    if (reconnectTimer) {
+      clearTimeout(reconnectTimer)
+    }
```

Comment:
It's [safe](https://www.w3.org/TR/2011/WD-html5-20110525/timers.html#dom-windowtimers-cleartimeout) to call `clearTimeout` with an undefined argument:

```suggestion
    // this is safe. it will not error even if the timer id is undefined or null
    clearTimeout(reconnectTimer)
```

Reply:
I did it more to show intent than because of safety. Though I am ok with doing this as well.